### ISSUE TITLE
Error while training yolov8

### ISSUE THREAD

**Issue by Ashlesha1995** [ultralytics/ultralytics#325]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

Hi,

I was traing my custom data on yolov8 using below code

![code_snipet](https://user-images.githubusercontent.com/42487151/212246420-c3ce66cf-8e29-49d2-b795-91d1f6211e88.png)

and i got this Error please help me!!

![error_yolov8](https://user-images.githubusercontent.com/42487151/212246547-47285ee7-d2d3-4893-ab66-185d1096fdf9.png)

Thanks advance,

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by Laughing-q** [comment]
@Ashlesha1995 hi, it could be the same issue as this one, can you check it? #234, #253

**Comment by DAVIDNYARKO123** [comment]
Make sure the right virtual environment which has ultralytics is selected. 

Try this after

`
from ultralytics import YOLO

# Load a model
model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

if __name__ == '__main__':
    # Use the model
    results = model.train(data="coco128.yaml", epochs=3)  # train the model
    results = model.val()  # evaluate model performance on the validation set
    results = model("https://ultralytics.com/images/bus.jpg")  # predict on an image
    success = YOLO("yolov8n.pt").export(format="onnx")  # export a model to ONNX format 

`

**Comment by Laughing-q** [comment]
@Ashlesha1995 I'm closing this issue as the same issue had been solved. Feel free to reopen if it still happens after you check the data. :)
